src/styles/aui/_aui.navigation-menu.scss

```diff
@@ -0,0 +1,427 @@
+@use '../quarks/quarks.variables';
```

Comment:
Deze lijn is overbodig.

Reply:
technisch gezien is deze lijn idd niet nodig, maar ik probeer de nieuwe sass standaard te hanteren. https://sass-lang.com/documentation/at-rules/import

Reply:
Voor mij hoeft het er niet per se uit, maar zie wel dat vorige versies van Node met deze package kunnen blijven werken.